Dockerfile

```
@@ -100,6 +100,11 @@ RUN printf "Start Jahia's installation...\n" \
         -i /usr/local/tomcat/conf/catalina.properties \
     && echo
 
+RUN jar xvf /usr/local/tomcat/webapps/ROOT/WEB-INF/lib/jahia-impl-*.jar META-INF/MANIFEST.MF \
+    && grep Implementation-Version META-INF/MANIFEST.MF | cut -f2 -d\ | cut -f1 -d- > version.txt \
+    && rm META-INF/MANIFEST.MF \
+    && echo Current Jahia Version : `cat version.txt`
+
```

May be something like will be great for avoiding extracting file, cleaning file and overall readability:
```suggestion
RUN unzip -p /usr/local/tomcat/webapps/ROOT/WEB-INF/lib/jahia-impl-*.jar META-INF/MANIFEST.MF \
    | awk '$1~/^Implementation-Version/ {print $NF}' > version.txt \
    && echo Current Jahia Version : `cat version.txt`
```

---

agree for the unzip, but the awk result does not give the same as the previous one, which removes everything after - 

maybe something like that :

```suggestion
RUN unzip -p /usr/local/tomcat/webapps/ROOT/WEB-INF/lib/jahia-impl-*.jar META-INF/MANIFEST.MF \
    | awk '$1~/^Implementation-Version/ {split($2,a,"-");print a[1]}' > version.txt \
    && echo Current Jahia Version : `cat version.txt`
```
?